diff --git a/Makefile b/Makefile
index a805a1b..6bf3bf8 100644
--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,16 @@
+# "prefix" path for installing "piqi" and "piqic" executables
+PIQI_PREFIX := /usr/local
+PIQI_OCAML_PREFIX :=
+PIQI_ROOT := $(shell pwd)
+PIQI_BUILD := $(PIQI_ROOT)/build
+PATH := $(PIQI_ROOT)/bin:$(PATH)
+PIQI_DIR := $(PIQI_ROOT)
+export PIQI_PREFIX PIQI_OCAML_PREFIX PIQI_ROOT PIQI_BUILD PATH PIQI_DIR
+
+SKIP-ulex = true
+SKIP-xmlm = true
+SKIP-easy-format = true
+
 include $(PIQI_ROOT)/make/Makefile.dirs
