--- msgpack-ocaml-coq8.3/ocaml/OMakefile.orig	2014-04-02 23:14:58.906965658 +0200
+++ msgpack-ocaml-coq8.3/ocaml/OMakefile	2014-04-02 23:16:31.409498823 +0200
@@ -26,7 +26,7 @@
 	num
 
 OCAML_WARN_FLAGS=$`(if $(equal $<,msgpackCore.ml),-annot,-w A -warn-error A ) #`
-OCAMLFLAGS=$(OCAML_WARN_FLAGS) -annot
+OCAMLFLAGS=$(OCAML_WARN_FLAGS) -w -44 -annot
 # ------------------------------
 # library
 # ------------------------------
